x/concentrated-liquidity/msg_server_test.go

```diff
@@ -231,6 +243,14 @@ func (suite *KeeperTestSuite) TestCollectIncentives_Events() {
 			expectedCollectIncentivesEvent:      3,
 			expectedMessageEvents:               4, // 1 for collect incentives, 3 for send messages
 		},
+		"error: three position IDs - not an owner": {
+			upperTick:                  DefaultUpperTick,
+			lowerTick:                  DefaultLowerTick,
+			positionIds:                []uint64{DefaultPositionId, DefaultPositionId + 1, DefaultPositionId + 2},
+			numPositionsToCreate:       2,
+			shouldSetupUnownedPosition: true,
+			expectedError:              cltypes.NotPositionOwnerError{},
```

Comment:
Shouldn't this error message contain the specific position ID and sender (using whichever of the two positions it errors on) as per the error type?

Reply:
I opted in for `ErrorAs` check which only checks the error type:
https://github.com/osmosis-labs/osmosis/blob/6270bc674f24f9352a5693efe3518b7d6f4542bc/x/concentrated-liquidity/msg_server_test.go#L196

The reason is that we run `SetupTest()` for every test case. As a result, `s.TestAccs[0]` gets reset on every iteration.

While checking that there are specific arguments formatted to the error is stricter than just the error type, I went with the faster approach to set it up since the marginal benefit of validating the formatting is insignificant IMO.